### PR TITLE
init deploy playbook

### PR DESCRIPTION
**Pull request by roman-doomitro**

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Servers
+
+1. [Nginx](https://nginx.org)
+2. [Puma](http://puma.io/)
+
+## Managers
+
+1. [RVM](https://rvm.io/)
+2. [NVM](https://github.com/creationix/nvm)
```
Comment (arfelio) [inline deploy/README.md]: why not rbenv?

Comment (blasterun) [inline deploy/README.md]: +1 for rbenv

**Comment by kiosan** [comment]
do we need hosting section?

File: deploy/README.md (hunk 1 above)
Comment (roman-doomitro) [inline deploy/README.md]: actually, i don't see a difference between rbenv and rvm, so if you provide any reason why should be rbenv or rvm is worse then rbenv, i will gladly replace rvm with rbenv

**Comment by roman-doomitro** [comment]
@kiosan it's just a basic section of what to use for deployment, in future i planned to improve this section of playbook with some examples and more information

File: deploy/README.md (hunk 1 above)
Comment (blasterun) [inline deploy/README.md]: `rbenv` better fits into 'shell' when you switching between projects its always know what version of ruby to use... and its more light. With `rvm` a had many problems with env configs

**Comment by kiosan** [comment]
let's merge this

**Pull request merged**